dev/ant_build/build-olt.xml

```diff
@@ -1,5 +1,5 @@
 <!--
- * Copyright (c) 2017 IBM Corporation and others.
+ * Copyright (c) 2018 IBM Corporation and others.
```

Comment:
Copyright should have both dates - original creation date and latest update date so:
    Copyright (c) 2017, 2018 IBM Corporation and others.
Same for the other files.

Reply:
Thanks for the catch. Fixed.